shared_requirements.txt

```diff
@@ -158,8 +158,8 @@ avro<2.0.0,>=1.10.0
 opencensus>=0.6.0
 opencensus-ext-threading
 opencensus-ext-azure>=0.3.1
-opentelemetry-api==0.17b0
-opentelemetry-sdk==0.17b0
+opentelemetry-api==1.0.0
```

Comment:
Why are we pinning an exact version?

Reply:
so far we have pinned since it was in beta and things were breaking randomly - let's keep it this way till monitor exporter is released and we come up with a plan on how to keep both versions identical